lib/hosting/hetzner_apis.rb

```diff
@@ -8,6 +8,32 @@ def initialize(hetzner_host)
     @host = hetzner_host
   end
 
+  def reset(server_id)
+    unless Config.hetzner_ssh_key
+      raise "hetzner_ssh_key is not set"
+    end
+
+    key_data = Config.hetzner_ssh_key.split(" ")[1]
+    decoded_data = Base64.decode64(key_data)
+    fingerprint = OpenSSL::Digest::MD5.new(decoded_data).hexdigest
+    formatted_fingerprint = fingerprint.scan(/../).join(":")
+    connection = Excon.new(@host.connection_string + "/boot/#{server_id}/linux", user: @host.user, password: @host.password,
+      headers: {"Content-Type" => "application/x-www-form-urlencoded"})
+    response = connection.post(body: "dist=Ubuntu 22.04.6 LTS base&lang=en&authorized_key=#{formatted_fingerprint}")
```

Comment:
Can we use same connection similar to `pull_ips`?
```suggestion
    connection = Excon.new(@host.connection_string, user: @host.user, password: @host.password, headers: {"Content-Type" => "application/x-www-form-urlencoded"})
    response = connection.post(path: "/boot/#{server_id}/linux", body: "dist=Ubuntu 22.04.6 LTS base&lang=en&authorized_key=#{formatted_fingerprint}")
```